src/GitExtensions.SolutionRunner/PluginSettings.cs

```diff
@@ -44,27 +44,27 @@ internal class PluginSettings : IEnumerable<ISetting>
         /// <summary>
         /// Gets current value of <see cref="ExecutablePathProperty"/>.
         /// </summary>
-        public string ExecutablePath => source.GetValue(ExecutablePathProperty.Name, ExecutablePathProperty.DefaultValue, t => t);
+        public string ExecutablePath => source.GetValue(ExecutablePathProperty.Name);

```

Comment:
What happens if user doesn't provide a value? 
Does the default value still apply?

Reply:
I fixed it :)